lib/moonshot/tools/asg_rollout.rb

```diff
@@ -65,6 +65,10 @@ def wait_for_in_service(new_instance)
 
           loop do
             instance_health = asg.instance_health(new_instance)
+            if instance_health.out_of_service?
```

Comment:
Oh, I misunderstood this as being ELB health. Maybe this is OK -- let me do some local testing when I have a chance.